react-native-libraries.json

```diff
@@ -7151,13 +7151,6 @@
     "ios": true,
     "android": true
   },
-  {
-    "githubUrl": "https://github.com/ospfranco/react-native-quick-sqlite",
-    "ios": true,
-    "android": true,
-    "unmaintained": true,
-    "alternatives": ["@op-engineering/op-sqlite"]
-  },
```

Comment:
Instead of removing, let's update the entry:
```
{
    "githubUrl": "https://github.com/margelo/react-native-quick-sqlite",
    "ios": true,
    "android": true,
    "macos": true,
    "unmaintained": true,
    "newArchitecture": true,
    "alternatives": ["@op-engineering/op-sqlite", "react-native-nitro-sqlite"]
}
```